src/main/java/io/confluent/connect/hdfs/HdfsSinkConnectorConfig.java

```diff
@@ -246,8 +251,15 @@
 
   }
 
+  public static Map<String, String> addDefaults(Map<String, String> props) {
+    ConcurrentMap<String, String> propsCopy = new ConcurrentHashMap<>(props);
+    propsCopy.putIfAbsent(STORAGE_CLASS_CONFIG, HdfsStorage.class.getName());
+    propsCopy.putIfAbsent(HdfsSinkConnectorConfig.FORMAT_CLASS_CONFIG, AvroFormat.class.getName());
+    return propsCopy;
+  }
+
   public HdfsSinkConnectorConfig(Map<String, String> props) {
-    this(CONFIG_DEF, props);
+    this(CONFIG_DEF, addDefaults(props));
```

Comment:
Last comment ... is `addDefaults` necessary anymore since these configurations now have defaults? Otherwise, looks great.